### ISSUE TITLE
Reference to a statement has sometimes no effect?

### ISSUE THREAD

**Issue by babig** [christianvoigt/argdown#46]
I don't understand the difference between the following two examples: 

```
# Referring to statements / equivalence classes

## Example 1

A target argument with one premise is supportet by two arguments,
 for support argument 2, the support relation is explicitly mentioned,
 for support argument 1, the support relation is estalished by reference
 to the target argument's premise.

Result: map with 3 arguments, as expected.

### target argument A1

<A1>

(1) [P1]
----
(2) [K1]

### supporter 1 for A1

<S1>

(1) [H1]
----
(2) [P1]

### supporter 2 for A1

<T1>

(1) [I1]
----
(2) [P1]
  +><A1>

## Example 2

The same as above, but without the second support argument.

Result: empty map. That the conclusion of the support argument belongs to the same
 equvalence class as the premise of the target argument seems to have no effect.

### target argument A2

<A2>

(1) [P2]
----
(2) [K2]

### supporter 1 for A2

<S2>

(1) [H2]
----
(2) [P2]
```

**Comment by christianvoigt** [comment]
good catch! This is caused by the MapBuilder plugin that selects which of the defined arguments and statements to put into the map. By default, nodes that do not have any relations (edges) are excluded from the map. You can change this behavior by using the following config file or by using the map settings in the online editor (you can toggle them in the map view if you click on the cog wheel icon): 

```javascript
{
  map: {
    excludeDisconnected: false
  }
}
```

Now the two arguments will appear, but the support relation is still missing. The MapBuilder seems to ignore equivalences between conclusions and premises if none of these statements is inserted as node into the map.

You can change the selection mode of the MapBuilder plugin to verify this. For this you can use the following config file or change the statement selection mode in the online editor's map settings:

```javascript
{
  map: {
    excludeDisconnected: false,
    statementSelectionMode: 'all'
  }
}
```

Now all statements of your debate are inserted as nodes into the map and the support relation suddenly appears. As a workaround until this is fixed, you can use the statement selection mode "titled", which will insert only statements with a title to the map. This gives you control over which statements you want to hide from the map.

**Comment by christianvoigt** [comment]
fixed in argdown-cli 0.9.1.
The VS Code extension still needs to be updated. But I am already closing this as it is fixed in argdown-map-maker.
